aspnetcore/blazor/hosting-models.md

```diff
@@ -323,14 +425,22 @@ Blazor Hybrid apps have full access to native client API capabilities via .NET n
 
 ### Web-based deployment
 
-Blazor Server and Blazor WebAssembly are deployed as web apps that are updated on the next app refresh.
+Apps that use either the Blazor Server or Blazor WebAssembly hosting model are deployed as web apps that are updated on the next app refresh.
```

Comment:
```suggestion
Blazor web apps are updated on the next app refresh from the browser.
```

Reply:
Modified it to caps for "Blazor Web Apps," as the name is a proper noun matching a project template name.

Reply:
I don't think we actually want to use the proper noun here. Not all Blazor web apps are created using the Blazor Web App template. For example, older Blazor Server and Blazor Web Assembly apps.